pygromos/files/gromos_system/ff/openforcefield2gromos.py

```diff
@@ -109,7 +110,7 @@ def convertAngles(self):
                 atomI=key[0]+1
                 atomJ=key[1]+1
                 atomK=key[2]+1
-                k = force.k.value_in_unit(u.kilojoule / (u.mole * u.radian ** 2))
+                k = force.k.value_in_unit(u.kilojoule / (u.mole * u.degree ** 2)) #TODO: proper conversion to quartic
```

Comment:
conversion is described in GROMOS manual in "18.1. Conversion of force constants: bond-stretching and bond-angle bending interactions"

Reply:
👍